src/Tabs.tsx

```diff
@@ -0,0 +1,24 @@
+interface TabsProps {
+  tabOptions: string[];
+  selectedTab: string;
+}
+
+export default function Tabs(props: TabsProps) {
+  const selectedButtonClasses =
+    "inline-flex items-center h-10 px-4 -mb-px text-sm text-center bg-transparent border-b-2 sm:text-base focus:outline-none dark:border-blue-400 dark:text-blue-300 whitespace-nowrap text-blue-600 border-blue-500";
```

Comment:
You need to extract this into css classes files. Otherwise the application gets really hard to manage.